rel/overlay/etc/default.ini

```diff
@@ -73,6 +73,9 @@ default_engine = couch
 ; on startup if not present.
 ;single_node = false
 
```

Comment:
the convention here is to use `;` for a blank line between commented out sections, can you update pls.